Dockerfile

```diff
@@ -8,4 +8,3 @@ ADD https://github.com/rancherio/os/releases/download/${RANCHEROS_VERSION}/vmlin
 ADD https://github.com/rancherio/os/releases/download/${RANCHEROS_VERSION}/initrd /dist/initrd
 
 ENTRYPOINT ["/scripts/lay-down-os"]
-CMD ["/dev/sda"]
```

Comment:
Didn't seem like this was used at all.  If we were to default to something, "LABEL=RANCHER_STATE" seems appropriate, but that is what lay-down-os already defaults to, so maybe a default in the Dockerfile isn't needed.